aws/app/lambda/templates/templates.js

```diff
@@ -169,43 +175,98 @@ exports.handler = async function (event) {
       parameters: parameters,
     };
     const command = new ExecuteStatementCommand(params);
-    return await dbClient
-      .send(command)
-      .then((data) => {
-        if (data.records && data.records.length > 0) {
-          return { data: parseConfig(data.records) };
+    try {
+      const data = await dbClient.send(command)
+      if (data.records && data.records.length > 0) {
+        let returnedData = parseConfig(data.records);
+        if ( method === "INSERT"){
+          returnedData = await createBearerToken(dbClient, returnedData.records[0].formID, false, params)
         }
-        return { data: data };
-      })
-      .catch((error) => {
-        console.error(
+        return {data: returnedData};
+      }
+      return {data: data};
+    }
+    catch(error){
+      console.error(
           `{"status": "error", "error": "${formatError(error)}", "event": "${formatError(event)}"}`
-        );
-        return { error: error };
-      });
+      );
+      return { error: error };
+    }
   }
 };
 
+
 const parseConfig = (records) => {
   const parsedRecords = records.map((record) => {
-    let formID, formConfig, organization;
+    let formID, formConfig, organization, bearerToken;
     if (!process.env.AWS_SAM_LOCAL) {
       formID = record[0].longValue;
       if (record.length > 1) {
         formConfig = JSON.parse(record[1].stringValue.trim(1, -1)) || undefined;
         organization = record[2].isNull || undefined;
+        bearerToken = record[3].isNull || undefined;
```

Comment:
Callz me crazy but where is the property (isNull) defined and what does it return?  A boolean or the truthy value?

Reply:
This seems to be part of the pg lib for a field in a particular record.

Reply:
Cool :). Wasn't sure what type the object was referencing (reviewing on github instead of local)